fonts/stylesheet.css

```diff
@@ -0,0 +1,48 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on September 12, 2020 */
+
+
+
+@font-face {
+    font-family: 'pt_serifbold';
+    src: url('ptserif-bold-webfont.woff2') format('woff2'),
+         url('ptserif-bold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+
+}
+
+
+
+
+@font-face {
+    font-family: 'pt_serifbold_italic';
+    src: url('ptserif-bolditalic-webfont.woff2') format('woff2'),
+         url('ptserif-bolditalic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+
+}
+
+
+
+
+@font-face {
+    font-family: 'pt_serifitalic';
+    src: url('ptserif-italic-webfont.woff2') format('woff2'),
+         url('ptserif-italic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+
+}
+
+
+
+
+@font-face {
+    font-family: 'pt_serifregular';
+    src: url('ptserif-regular-webfont.woff2') format('woff2'),
+         url('ptserif-regular-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6170621)